console/src/components/CompactHeader/CompactHeader.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import kind from '@enact/core/kind';
+import Button from '@enact/agate/Button';
+
+import css from './CompactHeader.less';
+import {getPanelIndexOf} from '../../App';
+
+const CompactHeader = kind({
+	name: 'CompactHeader',
+
+	propTypes: {
+		children: PropTypes.node,
+		isFor: PropTypes.string,
+		noExpandButton: PropTypes.bool,
+		onExpand: PropTypes.func
+	},
+
+	styles: {
+		css,
+		className: 'compactHeader',
+		publicClassNames: true
```

Comment:
It won't make much difference but I don't think we need to support CSS class name overrides in the app side. More often, it'd be better for the app to build their components to directly suit their needs rather than rely on the more complicate override system.